packages/tizen_rpc_port/example/server/lib/message_server.dart

```diff
@@ -0,0 +1,205 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs
+
+import 'package:tizen_rpc_port/tizen_rpc_port.dart';
+
+const String _tidlVersion = '1.9.1';
+
+enum _DelegateId {
+  notifyCallback(1);
+
+  const _DelegateId(this.id);
+  final int id;
+}
+
+enum _MethodId {
+  result(0),
+  callback(1),
+  register(2),
+  unregister(3),
+  send(4);
+
+  const _MethodId(this.id);
+  final int id;
+}
+
+abstract class _CallbackBase extends Parcelable {
+  _CallbackBase(this.id, this.once) {
+    sequenceId = sequenceNum++;
+  }
+
+  int id = 0;
+  bool once = false;
+  int sequenceId = 0;
+  static int sequenceNum = 0;
+
+  String get tag => '$id::$sequenceId';
+
+  @override
+  void serialize(Parcel parcel) {
+    parcel.writeInt32(id);
+    parcel.writeInt32(sequenceId);
+    parcel.writeBool(once);
+  }
+
+  @override
+  void deserialize(Parcel parcel) {
+    id = parcel.readInt32();
+    sequenceId = parcel.readInt32();
+    once = parcel.readBool();
+  }
+}
+
+abstract class ServiceBase {
```

Comment:
In practice, we named the class that communicates with the client app `ServiceBase`. In terms of design, `ClientProxy` may be appropriate, but it will be difficult to change.

Reply:
Ah, ok I understand your comments.
It seems to need to add `NotifyCallback` prefix.
I'll add it.
Thanks!

Reply:
Done.